easybuild/easyblocks/generic/intelbase.py

```diff
@@ -97,8 +94,8 @@ def __init__(self, *args, **kwargs):
         self.setup_local_home_subdir()
         self.clean_home_subdir()
 
-        if self.cfg._config['license'][0] is not None:
-            _log.deprecated('No old style license parameter, use license_file', '2.0')
+        if 'license' in self.cfg._config:
```

Comment:
just `if 'license' in self.cfg` is sufficient now that https://github.com/hpcugent/easybuild-framework/pull/1155 is merged in